engine/src/main/java/org/destinationsol/game/ship/hulls/HullConfig.java

```diff
@@ -74,6 +75,18 @@ public int getNrOfGunSlots() {
         return new ArrayList<>(data.gunSlots);
     }
 
+    public ParticleEmitter getParticleEmitter(int slotNr) {
+        return data.particleEmitters.get(slotNr);
+    }
+
+    public int getNrOfParticleEmitters() {
```

Comment:
Do we really need this method? We can just do `.getParticleEmitters().size()` instead.